package.json

```diff
@@ -21,8 +22,16 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
+    "eslint": "^3.15.0",
+    "eslint-config-airbnb": "^14.1.0",
```

Comment:
If we move to `eslint:recommended` we can get rid of all of these modules we don't use anyway. I see neither `import` nor `jsx` or `react` on the horizon for this project. If - if they would ever find their way in here, we can easly add this and change the `.eslintrc` accordingly.